.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
I know and it is already in Master, so I don't know why it happens to bebin the PR